src/utils/format-date.ts

```diff
@@ -0,0 +1,22 @@
+export type DateFormat = 'short' | 'long';
+
+export function formatDate(
+  date: Date | undefined,
+  format: DateFormat = 'short',
+  locale: string = 'en-US'
+): string {
+  if (!date) return '';
```

Comment:
I haven't seen where this is being consumed, but would it make sense to have this return `undefined` if the `date` is `undefined`?

Reply:
@jbuckner Since the output of this method is used for UI, in `render()`, I'm guessing that the empty string is the best result of a date that can't be formatted.

Reply:
Yeah potentially. That's why I was curious to see how it was being used, but this is fine 👍